src/main/scala/uk/gov/nationalarchives/tdr/api/service/FullHealthCheckService.scala

```diff
@@ -0,0 +1,18 @@
+package uk.gov.nationalarchives.tdr.api.service
+
+import slick.jdbc.PostgresProfile.api._
+import uk.gov.nationalarchives.tdr.api.db.DbConnection
+import uk.gov.nationalarchives.tdr.api.db.repository.TransferringBodyRepository
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class FullHealthCheckService {
+
+  def checkDbIsUpAndRunning(db: Database): Future[Unit] = {
```

Comment:
Do you need to pass the database in here? We're not passing the database in any other methods I think so it might be better to either create the `TransferringBodyRepository` directly in the class like
`val transferringBodyRepository = new TransferringBodyRepository(DbConnection.db)`

Or pass the repository into the service like we do with the others so the class declaration would  be `class FullHealthCheckService(transferringBodyRepository: TransferringBodyRepository)`

I would be tempted to do the first one so we don't have to pass that `Database` object round a lot but it's up to you.